### ISSUE TITLE
`imageio.plugins` not packaged

### ISSUE THREAD

**Issue by laggykiller** [Nuitka/Nuitka#2472]
- Nuitka version, full Python version, flavor, OS, etc. as output by *this exact* command.

  > python -m nuitka --version

```
1.9rc4
Commercial: None
Python: 3.11.5 (tags/v3.11.5:cce6ba9, Aug 24 2023, 14:38:34) [MSC v.1936 64 bit (AMD64)]
Flavor: CPython Official
Executable: D:\Downloads\imageio-test\venv\Scripts\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe (cl 14.3).
```

- How did you install Nuitka and Python

Python from python.org, exe installer

Nuitka from github factory version

- The specific PyPI names and versions

```
av==10.0.0
imageio==2.31.5
```

test.py
```
import imageio.v3

meta = imageio.v3.immeta('0.mp4', plugin='pyav')
print(meta)
```

Also copy in any mp4 video named `0.mp4` into the test.dist directory after compiling. Here is a small 15KB `0.mp4` you may use for testing: https://github.com/Nuitka/Nuitka/assets/61652821/0bca7043-90b3-4795-9c0d-354dd17a48b7

Compiling with `python -m nuitka --standalone --follow-imports --include-package=av test.py`

Running compiled version fails with:
```
Traceback (most recent call last):
  File "D:\DOWNLO~1\IMAGEI~1\TEST~1.DIS\test.py", line 3, in <module>
  File "D:\DOWNLO~1\IMAGEI~1\TEST~1.DIS\imageio\core\imopen.py", line 156, in imopen
ImportError: The `pyav` plugin is not installed. Use `pip install imageio[pyav]` to install it.
```

Adding `--include-package=imageio` would fix the problem.

While it is total expected that I have to add `--include-package=av` (because I did not import it in my code), I have to also add `--include-package=imageio` to make the compiled version work.

~Is this a feature or bug to ignore `imageio.plugins`? It only contains python codes for binding to pyav, pillow and other library. I think adding and ignoring `imageio.plugins` are both sensible options.~ On second thought, `imageio` without `imageio.plugins` is practically useless, as the whole idea of imageio is to provide interface to other library, it makes sense to include `imageio.plugins` by default

**Comment by kayhayen** [comment]
We have some explicitly listed.

```
- module-name: 'imageio.core.imopen'
  implicit-imports:
    - depends:
        - 'imageio.plugins.pillow_legacy'
        - 'imageio.plugins.freeimage'
        - 'imageio.plugins.simpleitk'
```

You can --include-module more of them. It will turn into a dependency bomb if we do more I think. We have yet to come up with a way of how to expose these plugin sets for choice to the user, even to make it aware that the issue even exists.

It seems that imageio has some specific error handling, that we might be able to hook for non-deployment mode at least.

**Comment by kayhayen** [comment]
So, here we go, pretty nice, with a bit of yaml config we go there:

```
Nuitka:INFO: Launching 'MiniImageIOPlugin.dist\MiniImageIOPlugin.exe'
Traceback (most recent call last):
  File "C:\Users\kayha\repos\Py2C\MINIIM~1.DIS\MiniImageIOPlugin.py", line 6, in <module>
  File "C:\Users\kayha\repos\Py2C\MINIIM~1.DIS\imageio\v3.py", line 253, in immeta    
  File "C:\Users\kayha\repos\Py2C\MINIIM~1.DIS\imageio\core\imopen.py", line 156, in imopen
RuntimeError: The `pyav` plugin is not installed. Use `--include-module [imageio.plugins.pyav]` with Nuitka to include it.
```

**Comment by kayhayen** [comment]
Only a bit of polishing, and this is usable. At some point, we might want to start avoiding to use RuntimeError but have a dedicated one in nuitka with `--deployment` not set, but I guess we are not there.

**Comment by kayhayen** [comment]
I think this was only forgotten to be closed and is solved in 1.8, definitely works in 1.9 right now it seems.
